### DISCUSSION TITLE
Incorporating and AI TTS for narating books that don't have an audiobook version.

### DISCUSSION THREAD

**Discussion by xxredxpandaxx** [advplyr/audiobookshelf#2044] [Ideas]
This is probably a pretty big feature to add but it would be great to be able to listen to books that don't have an audiobook version. While it wouldn't be perfect, AI TTS systems are getting quite good. I did a test with a couple different voices from Natural Reader and the results are definitely ok to listen to in my opinion. I haven't had a chance to test this with some of the opensource TTS models that are available yet but if they are close I think they would work just fine.

**Comment by advplyr** [comment]
There is a feature request open for this https://github.com/advplyr/audiobookshelf/issues/601

**Reply by xxredxpandaxx** [reply]
Opps. I searched TTS and nothing came up. Sorry about that!
